packages/project-starter/src/plugin.ts

```diff
@@ -199,7 +240,50 @@ const plugin: Plugin = {
         presencePenalty = 0.7,
       }: GenerateTextParams
     ) => {
-      return 'Never gonna make you cry, never gonna say goodbye, never gonna tell a lie and hurt you...';
+      // Implement real OpenAI fetch call without fallbacks
+      logger.info('Using OpenAI for TEXT_LARGE model');
+
+      const response = await fetch('https://api.openai.com/v1/chat/completions', {
```

Comment:
same comment here we should probably revert both of these - feel free to explain your thinking thanks!